edx/frontend-app-gradebook
# fix: Hard-coded link in Header component changed by LMS_BASE_URL env var

`index.jsx` file of Header component updated with change the hard-coded link of `https://www.edx.org` with a component property.
`index.jsx` file of root project updated. Added the environment variable `LMS_BASE_URL` to the parameters of the Header component


Issue: https://github.com/openedx/build-test-release-wg/issues/55

**What changed?**

- This updates the `index.jsx` file of the Header component with change the hard-coded link with the component property `logoDestination`.
- This updates the `index.jsx` file of root project with add the `LMS_BASE_URL` env var as `logoDestination` in Header component.

**Developer Checklist**

**Testing Instructions**

If you start your local development server for gradebook and you load a course, when you click in the header logo, then it should redirect to the LMS dashboard.

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:



FYI: @edx/masters-devs-gta
